Gruntfile.js

```diff
@@ -31,11 +31,58 @@ module.exports = function(grunt) {
           urls: ["http://localhost:8512/tests/SpecRunner.html"]
         }
       }
-    }
+    },
+    browserify: {
+      // browserify src/resolver.js -o dist/amd-resolver.js -s amd-resolver --dg false
+      options: {
+        browserifyOptions: {
+          detectGlobals: false,
+          standalone: "amd-resolver"
+        }
+      },
+      debug: {
+        options: {
+          browserifyOptions: {
+            debug: true
+          }
+        },
+        src: "src/resolver.js",
+        dest: "dist/amd-resolver.js"
+      },
+      release: {
+        src: "src/resolver.js",
+        dest: "dist/amd-resolver.js"
+      }
+    },
+    watch: {
+      browserify: {
+        files: "src/**/*.js",
+        tasks: "build-debug"
```

Comment:
This could be an array of `["browserify:debug", "jshint"]`, but I thought it better to simply run the `build-debug` task so as to prevent duplication of information.